site/macros.py

```diff
@@ -39,7 +39,7 @@ def iceberg_spark_runtime(spark="3.3", scala="2.12", version=__latest_iceberg_ve
     @env.macro
     def nessie_spark_extensions(spark="3.3", scala="2.12", version=__latest_nessie_version):
         return __maven_artifact(
-            "org.projectnessie",
+            "org.projectnessie.nessie-integrations",
```

Comment:
This will probably break docs until we release... Would it be worth moving doc change into a separate PR to be merged after the next release?

Reply:
_sigh_ yes, let me factor this to a separate PR.

Reply:
#6207